pkg/cosign/verify.go

```diff
@@ -84,6 +86,11 @@ type CheckOpts struct {
 
 	// SignatureRef is the reference to the signature file
 	SignatureRef string
+
+	// Identities is an array of Identity (Subject, Issuer) matchers that have
+	// to be met for the signature to ve valid.
+	// Supercedes CertEmail / CertOidcIssuer
+	Identities []v1alpha1.Identity
```

Comment:
This causes every user of `pkg/cosign` to depend on k8s; updating to a recent Cosign version thus adds 1.157 _million_ lines of code.

Please consider some other API design.

Reply:
zomg, I'm so sorry about that, didn't realize that:
https://github.com/sigstore/cosign/pull/1790

@mtrmac